go/run.go

```diff
@@ -217,15 +216,21 @@ func callMethod(name string, method reflect.Value, args []reflect.Value, params
 	reflectRes := executeMethod.Call([]reflect.Value{})
 
 	/* assuming we always have result, *ApiResponse, error */
-	output.Result = reflectRes[0].Interface()
-	apiResponseVar := reflectRes[1].Interface()
+	responseLength := len(reflectRes)
 	var apiResponse *sdk.APIResponse
-	if apiResponseVar != nil {
-		apiResponse = apiResponseVar.(*sdk.APIResponse)
-	}
-	callErr := reflectRes[2].Interface()
-	if callErr != nil {
-		output.Error = &ErrorStruct{Message: callErr.(error).Error()}
+	if responseLength == 3 {
+		output.Result = reflectRes[0].Interface()
+		apiResponse = reflectRes[1].Interface().(*sdk.APIResponse)
```

Comment:
please make sure that reflectRes[1].Interface() is not nil first - that's what the `apiResponseVar` above was used for.